homework.js

```diff
@@ -0,0 +1,124 @@
+//Q1 ->work
+const summation = function (number) {
+    let sum = 0;
+    for (let i = 0; i <= number; i++) {
+        sum = sum + i;
+    }
+    console.log(sum);
+}
+summation(5);
+console.log("------");
+//Q2 ->work
+const summationEven = function (number) {
+    let sum = 0;
+    for (let i = 0; i <= number; i++) {
+        if (i % 2 === 0) {
+            sum = sum + i;
+        }
+    }
+    console.log(sum);
+}
+summationEven(5);
+console.log("------");
+//Q3 ->work
+const avg1 = [8, 2, 2, 4];
+const avg = function (arr) {
+    let sum = 0;
+    let avg = 0;
+    for (let i = 0; i < arr.length; i++) {
+        sum = sum + arr[i];
+    }
+    console.log(avg = sum / arr.length);
+}
+avg(avg1);
+console.log("------");
+//Q4 ->ok but !! 
+const reverse1 = function (wordReverse) {
+    console.log(wordReverse.split("").reverse().join("").split(" ").reverse().join(" "));
```

Comment:
you dont need to do this twice, this also works

` return word.split('').reverse().join(''); `